### PR TITLE
Ranjitha/branch

### PR DESCRIPTION
**Pull request by ranjithavelusamy**
Branch page is updated

### CODE REVIEW

File: html/branch.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                            <svg class="side-nav__icon" version="1.1" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
+                                <path d="M16.4 4c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4c0 0.961 0.568 1.784 1.384 2.167-0.082 1.584-1.27 2.122-3.335 2.896-0.87 0.327-1.829 0.689-2.649 1.234v-4.121c0.825-0.38 1.4-1.209 1.4-2.176 0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4c0 0.967 0.576 1.796 1.4 2.176v7.649c-0.825 0.379-1.4 1.208-1.4 2.175 0 1.326 1.074 2.4 2.4 2.4s2.4-1.075 2.4-2.4c0-0.961-0.568-1.784-1.384-2.167 0.082-1.583 1.271-2.122 3.335-2.896 2.030-0.762 4.541-1.711 4.64-4.756 0.829-0.378 1.409-1.21 1.409-2.181zM6 2.615c0.764 0 1.384 0.619 1.384 1.385 0 0.764-0.62 1.385-1.384 1.385s-1.384-0.621-1.384-1.385c0-0.766 0.619-1.385 1.384-1.385zM6 17.385c-0.766 0-1.385-0.621-1.385-1.385 0-0.766 0.619-1.385 1.385-1.385 0.764 0 1.384 0.619 1.384 1.385 0 0.764-0.62 1.385-1.384 1.385zM14 5.385c-0.766 0-1.385-0.621-1.385-1.385 0-0.766 0.619-1.385 1.385-1.385 0.764 0 1.384 0.619 1.384 1.385 0 0.764-0.62 1.385-1.384 1.385z"></path>
+                                </svg>
+
```
Comment (Svpriyaa2808) [inline html/branch.html:54]: you can remove space

File: html/branch.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                            <span>About us</span>
+                        </a>
+                    </li>
+                </ul>
+                <div class="legal">
+                    &copy; 2024 by Git-Project
+                </div>
+            </div>
+            <div class="main-content">
+                <!--Here we will put our content, erase the filler class and div-->
```
Comment (Svpriyaa2808) [inline html/branch.html:72]: you can remove this line

File: styles/branch.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
+.main-content {
+    min-height: 100vh;
+    font-size: 2rem;
+}
+ 
+* .branch-page-content{
+    color: black;
+}
+ 
+
```
Comment (andrewkalumba) [inline styles/branch.css:10]: much spacing🤓

File: styles/branch.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+}
+ 
+ 
+.branch-page {
+    margin: 0;
+    padding: 8px;
+    border: 10px solid var(--color-primary-dark);
+    background-color: var(--color-primary-light);
+}
+
```
Comment (VaishaliGovindaraj) [inline styles/branch.css:17]: Remove whitespace

File: styles/branch.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+ 
+ 
+.branch-page {
+    margin: 0;
+    padding: 8px;
+    border: 10px solid var(--color-primary-dark);
+    background-color: var(--color-primary-light);
+}
+ 
+
```
Comment (andrewkalumba) [inline styles/branch.css:18]: spacing

**Review by andrewkalumba — APPROVED** [review]
looks great

File: styles/branch.css [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+ 
+.branch-command-explanation {
+    background-color: var(--color-primary-dark);
+    margin: 0px;
+    padding: 8px;
+    font-size: 1.5rem;
+
+ @media screen and (max-width:900px) {
+    font-size: 1.3rem;
+
```
Comment (TobiasKjernell) [inline styles/branch.css:83]: lets clear up this white space aswell

File: html/branch.html [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+                            <div class="branch-command-syntax">git merge main</div>
+                            <div class="branch-command-explanation">Merge the main branch into your current branch. Resolve any conflicts (if necessary) and stage them.</div>
+                            <div class="branch-command-syntax">git merge -- abort</div>
+                            <div class="branch-command-explanation">Exit from the merge process and return the branch to the state before the merge began.</div>
+                        </div>
+ 
+ 
+                    </div>
+                </div>
+                <!--Ends here-->
```
Comment (Svpriyaa2808) [inline html/branch.html:122]: you can remove this line too

**Review by VaishaliGovindaraj — APPROVED** [review]
Code looks good

**Review by TobiasKjernell — APPROVED** [review]
looks good when changes is done!

**Review by Svpriyaa2808 — APPROVED** [review]
good to go

**Pull request merged**